docs/portal.properties-Reference.md

```diff
@@ -285,3 +285,18 @@ This gene set will add the following in the query box:
 ```
 "BRCA genes" BRCA1: MUT=E1258D; BRCA2: HOMDEL MUT=NONSENSE MUT=NONSTART MUT=NONSTOP MUT=FRAMESHIFT MUT=SPLICE MUT=TRUNC;
 ```
+
+# Spark Configuration Settings
```

Comment:
We should add these properties to [portal.properties.EXAMPLE](https://github.com/cBioPortal/cbioportal/blob/master/src/main/resources/portal.properties.EXAMPLE)